frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -9,15 +9,20 @@ import {
 } from 'components/common/form';
```

Comment:
This form is missing styling, but the parcel search functionality is working:
1) search by pin
2) search by pid
3) search by address via geocoder.
4) search by dropping a marker on the map